docs/guidance/EnterpriseScaleAnalytics-SelfService.md

```diff
@@ -0,0 +1,117 @@
+
+This architectural paradigm shift ultimately leads to an increased velocity within the data platform as data consumers do no longer have to rely on a set of centralized teams or have to fight for prioritization of the requested changes. As smaller teams take ownership of the end-to-end integration workflow, the feedback loop between data provider and data consumer is much shorter and therefore allows for much quicker prioritization, development cycles and a more agile development process. Additionally, complex synchronization processes and release plans between teams are no longer required as the cross-functional Data Integration team has full visibility of the end-to-end technical stack as well as any implications that may arise due to the implemented changes. The team can apply software engineering practices to run unit and integration tests to minimize overall impact on consumers.
+
+In an ideal world, the data integration would be owned by the same team that owns the source systems. But the team should in general not only consist of data engineers that work on the source systems, but also of subject matter experts (SMEs) for these datasets, cloud engineers and data product owners. Such a cross-functional team setup reduces the communication required to teams outside and will be essential when developing the complete stack from infrastructure to actual data pipelines.
+
+Integrated datasets from source systems become the foundation of the data platform and will enable Data Product teams to further innovate on top of the business fact tables to eventually improve decision making and optimize business processes. Both Data Integration and Data Product teams should offer SLAs to consumers and ensure hat agreements are met. These SLAs can be related to data quality, timeliness, error rates, uptime and other tasks.
+
+### Summary
+
+The sections above are summarizing the scaling mechanisms within Enterprise-Scale Analytics that organizations can use to grow their data estate within Azure over time without running into well-known technical limitations. Both scaling mechanisms are helping to overcome different technical complexities and can be used in an efficient and simple way.
+
+The previous section promises agility and quick development cycles within the Data Integration and Data Product environments. However, this will only be possible if teams get the required access rights to develop independently and roll out new dataset versions and features over time in a self-service manner. How this can be achieved will be covered as part of the next section.
```

Comment:
Consider rephrasing? 

"In the previous section, we discussed strategies to enable scale; improve agility; and reduce development lifecycle within Data Integration and Data Product environments. However, this will only be possible if teams have appropriate access which enables self-serve. How this can be achieved will be discussed in the following section."